base-ubuntu-2204/Dockerfile

```diff
@@ -20,8 +20,9 @@ COPY ./files /
 
 RUN set -xeu && \
     ln -snf /usr/bin/bash /usr/bin/sh && \
-    install_packages busybox python3-pip supervisor xsltproc curl tree python-is-python3 && \
+    install_packages busybox python3-pip supervisor xsltproc curl tree python-is-python3 openssh-client openssh-server && \
     mkdir /opt/busybox && \
+    mkdir -p /run/sshd && \
```

Comment:
is it necessary? A program install via APT usually should be set up properly already.

Reply:
Will test

Reply:
Seems it's necessary. If not, throw error:

```
root@hadoop-master1:/# /usr/sbin/sshd -D
/etc/ssh/sshd_config line 123: Deprecated option RSAAuthentication
Missing privilege separation directory: /run/sshd
```

supervisor start filed.

Reply:
```
mkdir /run/sshd && chmod 0755 /run/sshd && \
mkdir /opt/busybox && busybox --install /opt/busybox
```